aten/README.md

```diff
@@ -241,3 +241,17 @@ of this processing occurs, it's helpful to look at the generated file
 ATen methods in a uniform manner.  This file is utilized by PyTorch
 which further extends the ATen interface with support for automatic
 differentation.
+
+#### Note [ATen preprocessor philosophy]
+
+ATen is defined to be simple to use, and one of the things this implies is
+that it should not be necessary to use preprocessor macros when using ATen;
+we would rather provide all symbols, even for functionality that is not
+available on the system ATen is running on.
+
+This means that internally inside ATen, whereas others might simply omit
```

Comment:
nit: I don't really know who "others" are -- do you mean other libraries?

Reply:
yep